livechat/utils/ws_client.py

```diff
@@ -24,13 +25,13 @@ class WebsocketClient:
     def __init__(self, url: str, timeout: int = 2):
         self.url = url
         self.timeout = timeout
-        self.websocket = None
+        self.websocket: WebSocket = None
```

Comment:
`self.websocket: WebSocket = None` is it correct here?

Reply:
It's a way of telling that this is a placeholder variable, but we expect it to be of type/class WebSocket. Otherwise mypy pre-hook would throw a lot of warnings. Not beautiful, but that's what happens when you want to make a dynamically typed language a static one 🤷